setup.md

```diff
@@ -9,31 +9,37 @@ You need to install OpenRefine and download a data file to follow this lesson.
+This lesson has been tested with all versions of OpenRefine up to the latest tested version, 3.4.
 
-If you are using an older version, it is recommended you upgrade to the latest tested version. 
+Packages are available on <https://openrefine.org/download.html> for Windows, macOS, and Linux.
+Please download the latest stable version, choosing the "kit" for your operating system.
+Current versions of the "Windows kit with embedded Java" and "Mac kit" include everything you need to run OpenRefine.
+The "Linux kit" and traditional "Windows kit" require a "Java Runtime Environment" (JRE) installed on your system (see notes below).
 
-There are versions for Windows, macOS and Linux.
+If you are using an older version of OpenRefine, it is recommended you upgrade to the latest tested version. 
 
 Please follow the installation instructions in the OpenRefine User Manual: [Installation Instructions](https://docs.openrefine.org/manual/installing)
 
 Notes:
-* When you download OpenRefine for Windows or Linux from the address above, you are downloading a zip file. To install OpenRefine you unzip the downloaded file wherever you want to install the program. This can be to a personal directory or to an applications or software directory - OpenRefine should run wherever you put the unzipped folder. The location has to be a "local" drive as problems have been reported trying to run OpenRefine from a Network drive.
-* OpenRefine is a Java application, and you need to have a 'Java Runtime Environment' (JRE) installed on your computer to run OpenRefine. Version 3.4 of OpenRefine for Windows has a version with embedded Java which you can use without doing a separate installation if the Java Runtime Environment.  If you are on Linux, or use Windows and do not want to use OpenRefine with embedded Java, and you don’t already have the Java Runtime Environment installed, then you can download and install from [http://java.com](http://java.com) by going to the site and clicking "Free Java Download".
-* OpenRefine does not support Internet Explorer or Edge. Please use [Firefox](https://www.mozilla.org/firefox/new/), [Chrome](https://www.google.com/chrome/) or [Safari](https://www.apple.com/safari/) instead.
+* When you download OpenRefine for Windows or Linux from the address above, you are downloading an archive file (zip or tar). To install OpenRefine unzip the downloaded file to a permanent location on your computer. This can be to a personal directory or to an applications or software directory - OpenRefine should run wherever you put the unzipped folder. The location has to be a "local" drive as problems have been reported trying to run OpenRefine from a Network drive.
+* In OpenRefine versions 3.4+ the options "Windows kit with embedded Java" and "Mac kit" include Java as part of the package. You **do not** need to install Java if you use one of these kits. This is the preferred method on Windows and Mac systems.
```

Comment:
I think the version info in this sentence is redundant (the Mac version has always had Java embedded and the "Windows kit with embedded Java" has embedded java by definition no matter what the version).

Suggest this could be amended to:
"The options "Windows kit with embedded Java" and "Mac kit" include Java as part of the package. You **do not** need to install Java if you use one of these kits. This is the preferred method on Windows and Mac systems."

Reply:
Makes sense to me.